colossalai/shardformer/flashattention/flash_attention_opt.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+from torch.distributed import ProcessGroup
+from xformers.ops import memory_efficient_attention as me_attention
+
+from colossalai.shardformer.layer.parallel_module import ParallelModule
+
+__all__ = ['FlashAttention']
+
+
+class FlashAttention(ParallelModule):
```

Comment:
Is this one only for OPT or it can be used for other models? If it is only for OPT, it should be put in the following file.

```
- shardformer
   - modelling
      - opt.py
```